docs/tutorials/code-organization.md

```diff
@@ -47,7 +47,7 @@ The modules folder is where you will add all of your project-specific modules. W
 
 ![screenshot of the open modules folder](../images/sec2-1b-modules-folder.png)
 
-The topmost of these is the `@apostrophecms` folder. Any modules in this folder will implicitly `improve` the core Apostrophe module of the same name. Any module which then `extends` that core module will also have access to these improvements. We will come back to `improve` and `extend` later in this tutorial. From within your project, you can see what modules are part of core by navigating to the `node_modules/apostrophe/modules/@apostrophecms` folder. *Note*: there is a `node_modules/@apostrophecms` folder. This is used for additional extensions added from your package manager, not for the core modules. As a rule, you should not add any custom module into any of the `node_modules` folders. Any code should be added from hosted packages.
+The topmost of these is the `@apostrophecms` folder. Any modules in this folder will implicitly `improve` the core Apostrophe module of the same name. Any module which then `extends` that core module will also have access to these improvements. We will come back to `improve` and `extend` later in this tutorial and you can read more in our [documentation section](/guide/module-configuration-patterns.html). From within your project, you can see what modules are part of core by navigating to the `node_modules/apostrophe/modules/@apostrophecms` folder. *Note*: there is a `node_modules/@apostrophecms` folder. This is used for additional extensions added from your package manager, not for the core modules. As a rule, you should not add any custom module into any of the `node_modules` folders. Any code should be added from hosted packages.
```

Comment:
Tiny thing and not new, but the "s" in "extends" should not be part of the code font.

Reply:
"As a rule, you should not add any custom module into any of the `node_modules` folders. Any code should be added from hosted packages."

This sentence is confusing.

How about:

"As a rule, you should never touch anything in `node_modules`, unless your goal is to publish your own module to `npm` to share it with the world, in which case see [some tutorial here]"

Reply:
Changed and slightly altered. No real tutorial to link (yet).